app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,7 @@
     <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Why did you remove READ_EXTERNAL_STORAGE?

Reply:
Because granting WRITE_EXTERNAL_STORAGE permission implicitly grants permission to read the external storage as well.

Source: (https://developer.android.com/training/data-storage/files#ExternalStoragePermissions)